go/ngql/query.go

```diff
@@ -37,29 +37,33 @@ const (
 
 func constructQueryType(rootValue types.Value, tm *typeMap) *graphql.Object {
 	rootNomsType := rootValue.Type()
-	rootType := nomsTypeToGraphQLType(rootNomsType, false, tm)
+	rootType := NomsTypeToGraphQLType(rootNomsType, false, tm)
 
 	return graphql.NewObject(graphql.ObjectConfig{
 		Name: rootQueryKey,
 		Fields: graphql.Fields{
 			rootKey: &graphql.Field{
 				Type: rootType,
 				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
-					return maybeGetScalar(rootValue), nil
+					return MaybeGetScalar(rootValue), nil
```

Comment:
Not super happy about exposing `MaybeGetScalar`, but I need to. If you can suggest a better name then great